brian_cheung/conditionals.md

```diff
@@ -27,13 +37,30 @@ Create a Ruby method called `temperature_status` that takes three arguments: the
   - If the air conditioner is non-functional and the current temperature is below the the desired temperature, print the string "Fix the A/C whenever you have the chance. It's cool."
 
 ```ruby
-# write your code here
+def temperature_status(current_temp, functioning, desired_temperature)
+	if (current_temp > desired_temperature) && functioning == true
+		puts "Turn on the A/X please."
+	elsif (current_temp > desired_temperature) && functioning == false
+		puts "Fix the A/C now! It's hot!"
+	elsif (current_temp < desired_temperature) && functioning == false
+		puts "Fix the A/C whenever you have the chance. It's cool."
+	else
+		puts "You have run into an error or there are no reponse available"
+	end
+end
 ```
 
 ## Palindrome Checker
 
 Create a Ruby method that takes a word or phrase and checks whether or not it's a palindrome (reads the same backward as forward).
 
 ```ruby
-# write your code here
+def palindromeTest(phrase)
+	 if phrase == phrase.reverse
```

Comment:
Ruby generally uses snake case (`palindrome_test`) over camel case (`palindromeTest`). You did that nicely with your variables above, but you want to make sure you do it everywhere.